samples/sensor/qdec/sample.yaml

```diff
@@ -0,0 +1,15 @@
+sample:
+  description: Usage quadrature decoder sensor
+  name: qdec_sensor
+tests:
+  sample.sensor.qdec_sensor:
+    tags: sensors
+    platform_allow: nucleo_f401re
+    timeout: 10
+    harness: console
+    harness_config:
+      type: multi_line
+      ordered: true
+      regex:
+        - "Quadrature decoder sensor test"
+        - "Counter value = 0"
```

Comment:
is it guaranteed to always start with zero?

Reply:
At least on ST devices the answer should be yes, but since I agree this is not general and also according to the previous comment, I removed this part